tasks/skit-sleep/skit-sleep.yaml

```diff
@@ -0,0 +1,22 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
```

Comment:
can you refactor this to remove the skit and app references from the params and names